gateway/api/serializers.py

```diff
@@ -138,9 +138,9 @@ def create(self, validated_data):
         pass
 
 
-class RunExistingJobSerializer(serializers.ModelSerializer):
+class RunAndRunExistingJobSerializer(serializers.ModelSerializer):
```

Comment:
nit: this name is a bit clunky, would it make sense to simplify to:

```suggestion
class RunJobSerializer(serializers.ModelSerializer):
```

Reply:
Fixed in [f824b4c](https://github.com/Qiskit-Extensions/quantum-serverless/pull/1286/commits/f824b4ca34fa4d379f6220fc7e8922f6b4c40026)